parts/appendices/B.fodt

```diff
@@ -13551,10 +13668,166 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Reduced execution time for parallel AMG/CPR simulations. Allowed use of AMG with command line option --matrix-add-well-contributions=false (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5182" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5182</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">The code has been updated to only issue warnings the first time the transmissibilities are calculated thus removing spurious and duplicate warnings that can occur when using load balancing (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5351" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5351</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Nearly all exceptions thrown when computing wells will not abort the simulator but result in a timestep chop. Hence those should not be counted as errors and be reported in the PRT file. The simulator now only reports problems for exceptions thrown when updating the well potentials (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5423" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5423</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">The well economic (WECON) minimum rates are no longer applied if the well is subject to a zero group rate target or constraint (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5424" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5424</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Directly output the SGMAX, SHMAX, SOMAX, SOMIN and SWHY1 arrays rather than the KRNSW_GO, KRNSW_OW, PCSWM_GO, PCSWM_OW and SOMAX arrays to the Restart file for cases with relative permeability and/or capillary pressure hysteresis (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5402" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5402</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Sphinx documentation has been added for the opm.simulators.BlackOilSimulator Python module (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5243" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5243</text:a>). </text:p>
```

Comment:
Maybe a link to the documentation would be nice. https://opm.github.io/opm-python-documentation/master/index.html